src/script/miniscript.h

```diff
@@ -1161,6 +1161,9 @@ struct Node {
         return true;
     }
 
+    //! Whether no satisfaction exists for this node.
+    bool IsNotSatisfiable() const { return !GetStackSize(); }
```

Comment:
In d4d3fcc16ccae4755604bb3a9a6a45542b231dcc "miniscript: treat all unsatisfiable miniscripts as insane"

Since this is only used negated, I think it would be better to make this `IsSatisfiable`

```suggestion
    bool IsSatisfiable() const { return GetStackSize().has_value(); }
```